text/0007-weekly-inspo.md

```diff
@@ -0,0 +1,289 @@
+the dev team will make a pull request to the `xplr` repository containing something 
+that inspired them. Their inspiration can be anything from a cool CSS property, an 
+interesting design, quirky libraries, or exciting frameworks. During our dev meeting, 
+we will talk about the stuff we found and at the end of the discussion, update a 
+`CHANGE_LOG` to keep a weekly timeline of the inspo we added.
+
+The pull request will be in the form of a markdown file that contains a description of 
+the inspo, why they think it's interesting, and how we could potentially use it in the 
+future. The file must also include metadata to help tag and catalog who the inspo came 
+from and what category the inspo belongs to.
+
+All these markdown files will accumulate into a blog site that is easily accessible to
+anyone outside of the dev team to read. The site will be searchable, filterable, and 
+creatively designed to show off our inspo in an unique fashion.
+
+Ultimately, the purpose of `xplr` is to constantly expose our dev team to the greater 
+world of dev and to familarize everyone with documentation, the PR process, and 
+exploration.
+
+## Why are we doing this?
+
+One of the things that we really stress at Creative Labs is having taste and 
+inspiration. Inspo is a big part of creativity and allows us to create better work. 
+Although inspo is usually something that is associated with the design side, exposing 
+ourselves to different dev concepts would make us better developers and empower us to 
+challenge our designers to make even more creative work.
+
+## What is the high level design/implementation?
+
+`xplr` can be built in an entirely self-contained environment. `xplr` will be built
+as a `Next.js` application that will be deployed to Netlify. There isn't a need for
+any servers as `Next.js` comes with an `api` that Netlify will deploy as a AWS function
+for us.
+
+### Organization
+
+The `xplr` repository will have a standard `Next.js` structure. On top of the standard
+configuration, we will also need to have a `_posts` and `_team` directory to keep 
+track of each inspo post and each team member. See [Appendix](#organization-(expanded))
+for more details on organization.
+
+### Verification
+
+In order to reduce the chance of introducing bugs into our application. We will need
+to write some custom linters to test the efficacy of the our posts.
+
+**Posts Linter**
+
+* Check that the author exists in the team
+* `title` is identical to the title of the article
+* `slug` is identical to file name
+* Check that `tag` is delimited white spaces and is an available tag in the'
+  `_posts/tags.yml` file
+
+**Team Members Linter**
+
+* Check if `name` is capitalized
+* Check that the file name is the `name` but in `kebab-case`
+
+**Markdown Linter**
+
+* Make sure all markdown files are only 80 characters long
+* Warnings for spelling would be a nice to have
+* Hopefully we can find a linter out there for this already
+
+### Search
```

Comment:
Since we intend this to be a blog of sorts that people can publicly discover on the interwebs as well, it may be a good idea to have a sitemap for SEO n all that. Haven't tried it out, but a quick googlin revealed this https://www.npmjs.com/package/nextjs-sitemap-generator which looks easy enough to use.

Reply:
Ooo.. hmmm I think this would be a nice to have feature!